app/controllers/microposts_controller.rb

```diff
@@ -1,4 +1,5 @@
-class MicropostsController < ApplicationController
+require 'api/api_controller'
+class MicropostsController < ApiController
```

Comment:
you have to have 2 microposts controller:
1 common as it was
2 separated in the API folder only with index, show (yep, it will duplicate code, but you will have 2 views)